module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -644,6 +645,40 @@ protected function getHoldingDetailsForItem($holding): array
         );
     }
 
+    /**
+     * Support method for getHolding() -- return an array of item-level details from
+     * both FOLIO holdings and item records.
+     *
+     * Depending on where this method is called, $locationId will be the holdings record
+     * location (in the case where no items are attached to a holding) or the item record
+     * location (in cases where there are attached items).
+     *
+     * @param string $locationId     Location identifier from FOLIO
+     * @param array  $holdingDetails Holding details produced by getHoldingDetailsForItem()
+     *
+     * @return array
+     */
+    protected function getItemFieldsFromLocAndHolding(
+        string $locationId,
+        array $holdingDetails,
+    ): array {
+        $locationData = $this->getLocationData($locationId);
+        $locationName = $locationData['name'];
+        $locationCode = $locationData['code'];
```

Comment:
We need the $locationName variable since it is used in multiple places below, but would it be more concise to directly reference $locationData below instead of creating $locationCode and $locationIsActive, since they're never reused and don't really make a big difference to readability?

Reply:
Minor matter of style, and I'm happy with either direction. I removed those under-utilized variables.